src/migration-wizard/steps/connect/sources-table/actions/DownloadOvaAction.tsx

```
@@ -0,0 +1,69 @@
+export const DownloadOvaAction: React.FC<DownloadOvaAction.Props> = (
+  props
+) => {
+  const { 
+    sourceId, 
+    sourceName, 
+    isDisabled = false, 
+  } = props;
+  
+  const discoverySourcesContext = useDiscoverySources();
+  const [isDownloading, setIsDownloading] = useState(false);
+  const [url, setUrl] = useState(discoverySourcesContext.getDownloadUrlForSource(sourceId));
```

ESLint

[error] Replace `discoverySourcesContext.getDownloadUrlForSource(sourceId)` with `⏎····discoverySourcesContext.getDownloadUrlForSource(sourceId),⏎··`

(prettier/prettier)